src/blurhash/__init__.py

```diff
@@ -34,3 +41,37 @@ def encode(image_file, x_components, y_components):
         raise ValueError('Invalid x_components or y_components')
 
     return _ffi.string(result).decode()
+
+def decode(blurhash, width, height, punch = 1, mode = 'RGBA'):
+
+    if not mode in _pixel_modes:
+        raise ValueError("Invalid value for argument mode, must be either 'RGB' or 'RGBA'")
+
+    channels = _pixel_modes[mode]
+    blurhash_str = _ffi.new('char[]', bytes(blurhash, "utf-8"))
+
+    if not _lib.is_valid_blurhash(blurhash_str) :
+        raise ValueError("{} is not a valid blurhash".format(blurhash))
+
+    width_int = _ffi.cast('int', width)
+    height_int = _ffi.cast('int', height)
+    punch_int = _ffi.cast('int', punch)
+    channels_int = _ffi.cast('int', channels)
+
+    pixel_array = _ffi.new('uint8_t[]', width * height * channels)
+
+    result = _lib.create_pixels_from_blurhash(blurhash_str, width_int, height_int,
+                                    punch_int, channels, pixel_array)
+
+    if result == -1 :
+        raise Exception("Failed to decode blurhash {}".format(blurhash))
+
+    pixels_buffer = _ffi.buffer(pixel_array, width * height * channels)
+    image = Image.frombuffer(mode, (width, height), pixels_buffer)
+
+    return image
+
+def is_valid_blurhash(blurhash) :
```

Comment:
Prefix the name of this function with an underscore since it seems that this is not meant to be part of the public interface of this library. If you think that this should be part of it, then it makes sense to include it in  `__all__`.

Reply:
@lautat , yes this is supposed to be a public-function, added it in `__all__`, this function allows anyone to check if the blurhash they have is valid or not without calling `decode` function.